superset-frontend/src/components/DvtDragCard/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Icon from '../Icons/Icon';
+import {
+  StyledDvtCard,
+  StyledDvtCardCard,
+  StyledDvtCardIcon,
+  StyledDvtCardLabel,
+} from './dvt-drag-card.module';
+
+export interface DvtDragCardProps {
+  label: string;
+  value: string;
+  icon: string;
```

Comment:
label ve icon ikise ui görüntü o tamam ama value bu bir item seçmek gibi düşünebilirsin.

api bir object item bu alacağız için

value: any; olabilir